### PR TITLE
Add new status `pre-liquidated` to DataCenterAssetStatus

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/ralph/data_center/migrations/0025_auto_20170510_1048.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```
Comment (mkurek) [inline src/ralph/data_center/migrations/0025_auto_20170510_1048.py]: keep only one migration :)

File: src/ralph/data_center/models/choices.py [hunk 2]
```diff
@@ -11,6 +11,7 @@ class DataCenterAssetStatus(Choices):
     free = _('free')
     damaged = _('damaged')
     liquidated = _('liquidated')
+    pre_liquidated = _("pre liquidated")
```
Comment (mkurek) [inline src/ralph/data_center/models/choices.py]: move it to the end

**Review by mkurek — CHANGES_REQUESTED** [review]

File: src/ralph/data_center/tests/test_models.py [hunk 3]
```diff
@@ -493,3 +493,20 @@ def setUp(self):
...
+        """
+        Tests that adding choice keeps ids the same (choices should be
+        appended)
+        """
+        statuses = DataCenterAssetStatus()
+        self.assertEqual(
+            statuses,
+            [
+                (1, 'new'), (2, 'in use'), (3, 'free'), (4, 'damaged'),
+                (5, 'liquidated'), (6, 'pre liquidated'), (7, 'to deploy'),
```
Comment (andrzej-jankowski) [inline src/ralph/data_center/tests/test_models.py]: `pre liquidated` should be the last one... ?

**Review by andrzej-jankowski — CHANGES_REQUESTED** [review]

File: src/ralph/data_center/tests/test_models.py (hunk 3 above)
Comment (ghost) [reply src/ralph/data_center/tests/test_models.py]: i copied it from time before correct migrations ;P

**Review by andrzej-jankowski — APPROVED** [review]

**Review by mkurek — APPROVED** [review]

**Pull request merged**